snapshot/lib/snapshot/reports_generator.rb

```diff
@@ -59,7 +59,9 @@ def available_devices
         'iPhone6' => "iPhone6 (4.7-Inch)",
         'iPhone5' => "iPhone5 (4-Inch)",
         'iPhone4' => "iPhone4 (3.5-Inch)",
```

Comment:
People still might use _snapshot_ for iOS 9, so the HTML reporter should still support them. We just removed it from the default `Snapfile`, since new projects will not find it useful